modules/virt-enabling-usb-host-passthrough.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * virt/advanced_vm_management/virt-configuring-usb-host-passthrough.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-enabling-usb-host-passthrough_{context}"]
+= Enabling USB host passthrough
+
+You can enable USB host passthrough at the cluster level.
+
+You specify a resource name and USB device name for each device you want to add and then assign to a virtual machine (VM). You can allocate more than one device, each of which is known as a `selector` in the HyperConverged (HCO) custom resource (CR), to a single resource name. If you have multiple, identical USB devices on the cluster, you can choose to allocate a VM to a specific device.
+
+.Prerequisites
+
+* Enabling the passthrough requires cluster administrator privileges.
```

Comment:
suggestion to keep similar wording to other procedures in the docs:
```suggestion
* You have access to an {product-title} cluster as a user with the `cluster-admin` role.
```

Reply:
Thanks!